test/features/user_records_an_application_test.rb

```diff
@@ -2,21 +2,23 @@
 
 class UserRecordsAnApplication < ActionDispatch::IntegrationTest
   def test_user_creates_an_application
+    user = create(:person)
+    page.set_rack_session(user_id: user.id)
     visit dashboard_path
     click_link_or_button 'new_application'
     fill_in 'application_company', :with => "Basecamp"
     fill_in 'application_location', :with => "Chicago, IL"
     fill_in 'application_url', :with => "http://basecamp.com/jobs"
     fill_in 'application_applied_on', :with => Date.today
-    select 'open', :from => 'application_status'
+    select 'applied', :from => 'application_status'
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.